### ISSUE TITLE
User data in request body

### ISSUE THREAD

**Issue by sarahschuetz** [born05/craft-sentry#28]
Is there a way to prevent the body of the request being sent to Sentry for specified endpoints?
I would like to avoid sending user information to sentry when for example the registration form is not working. I know I can manipulate the data in Sentry, so it is not shown, but ideally it would not even be sent to Sentry.

**Comment by roelvanhintum** [comment]
@sarahschuetz, there is an `anonymous` option, that prevents sending info of the logged in user. Other than that there is [filtering](https://docs.sentry.io/platforms/php/configuration/filtering/#using-platformidentifier-namebefore-send-) support in Sentry that could be implemented, if that is what you are looking for.

**Comment by sarahschuetz** [comment]
@roelvanhintum yes, I guess filtering is what I'm looking for. To use this hook however, I need a custom module or can I configure it with the plugin somehow?

**Comment by roelvanhintum** [comment]
This plugin currently doesn't have any support, but feel free to make a pull-request. Otherwise it would involve a custom module.

**Comment by sarahschuetz** [comment]
Okay, I wil have a look at it. Thanks!

**Comment by sarahschuetz** [comment]
@roelvanhintum added a pull-request. Would be great if you can have a look at it :)

**Comment by sarahschuetz** [comment]
Hi @roelvanhintum!
Did you have time to take a look at my pull-request?
